css/style.css

```
@@ -0,0 +1,68 @@
+body {
+    font-family: BlinkMacSystemFont, 'Segoe UI', 'Roboto';
+    font-size: 18px;
```

```suggestion
  font-size: 18px;
```

fix the indentation from tab to two spaces - more visually appealing, and a better practice for writing code
the font-family attribute should have a generic option such as sans-serif - to ensure support to all users